SIS_import/sis_update_url.rb

```diff
@@ -130,13 +135,12 @@ def processTermCourses(mPathwayTermSet,esbToken, outputFile)
 			outputFile.write("for term SIS_ID=#{term["sis_term_id"]} and Canvas term id=#{termId}\n")
 
 			# Web Service call
-			json_data = Canvas_APICall("#{$canvasUrl}/api/v1/accounts/1/courses?per_page=100&enrollment_term_id=#{termId}&published=true&with_enrollments=true",
-			                  "Bearer #{$canvasToken}",
+			json_data = Canvas_APICall("#{$canvasUrl}/api/v1/accounts/1/courses?per_page=#{$page_size}&enrollment_term_id=#{termId}&published=true&with_enrollments=true",
```

Comment:
How about a variable or method to construct the prefix so "#{$canvasUrl}/api/v1" doesn't have to be repeated?